public/app/themes/clarity/functions.php

```diff
@@ -81,6 +81,7 @@
 require_once 'inc/content-filter/search-query.php';
 require_once 'inc/content-filter/search.php';
 require_once 'inc/enqueue.php';
+require_once 'inc/environment-notice.php';
```

Comment:
I'm not sure if this was intentionally removed, I've assumed it was a mistake and added it back.

Reply:
Ah ... thanks @wilson1000 !